app/components/embed/media_with_companion_windows_component.html.erb

```diff
@@ -43,33 +43,33 @@
 
   <div class="media-component-body">
     <aside class="left-drawer collapse" data-media-target="leftDrawer" id="left-drawer">
-      <nav class="vert-tabs">
-        <button data-action="click->media#displayMetadata" data-media-target="leftButton" class="active" aria-label="About this item" aria-controls="about">
+      <nav class="vert-tabs" role="tablist">
+        <button data-action="click->media#displayMetadata" data-media-target="leftButton" class="active" aria-label="About this item" aria-controls="about" aria-selected="true" role="tab">
```

Comment:
Can you please add the logic to change the `aria-selected` when one of the other tabs is pressed?

Reply:
Done.